tasks/update_go.py

```diff
@@ -120,15 +126,106 @@ def _handle_file(path: str, patterns: Dict[re.Pattern, str], expected_match: int
     with open(path, "w") as writer:
         writer.write(content)
 
+def _check_appgate():
+    return
 
-@task(
-    help={
-        "version": "The version of Go to use.",
-        "check_archive": "If specified, download archive and check the SHA256.",
-        "warn": "Don't exit in case of matching error, just warn.",
-    }
-)
-def update_go(ctx: Context, version: str, check_archive: Optional[bool] = False, warn: Optional[bool] = False):
+# maybe this could be done with yq but it removes a lot of the formatting
+# https://github.com/mikefarah/yq/issues/515
+def _add_wait_for_tests(branch: str):
+    import re
+    trigger = """project: DataDog/datadog-agent"""
+    # load bearing yaml whitespace :sigh:
+    with_branch = f"""{trigger}
+    branch: {branch}"""
+    with open(".gitlab-ci.yml", "r") as f:
+        contents = f.read()
+    contents = re.sub(trigger, with_branch, contents, count=1)
+    with open(".gitlab-ci.yml", "w") as f:
+        f.write(contents)
+
+# TODO: not sure how to run this before merging
+def _remove_wait_for_tests(branch: str):
```

Comment:
It would be nice to have this run with a main build so we didn't need another commit, I'm not completely sure how to do that right now but it may be possible with a label and the github api